aspnetcore/fundamentals/logging/index.md

```diff
@@ -898,7 +901,17 @@ The `ILogger` implementation category name is typically the logging source. For
 
 [!code-csharp[](index/samples/3.x/CustomLogger/ColoredConsoleLogger/ColoredConsoleLogger.cs?name=snippet)]
 
-The preceding code creates a logger instance per category name.
+The preceding code:
+
+* Creates a logger instance per category name.
+* Checks `logLevel == _config.LogLevel` in `IsEnabled`, so each `logLevel` has a unique logger. Generally, loggers should also be enabled for all higher log levels:
```

Comment:
> Checks `logLevel == _config.LogLevel` in `IsEnabled`
This doesn't match the code and the code snippet anymore, right?

Reply:
@JuergenGutsch I reverted to your original code. The snippets are imported into the doc.

Reply:
@JuergenGutsch  oops, forgot to push  that change. It's fixed now